chall06/mlink.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+import fileinput
+import sys
+
+def error(error_message, filename):
+    print(f'{sys.argv[0]}: {filename}: {error_message}')
```

Comment:
I'm not sure creating a function error for that is really useful.

would have been better to use a built-in function like for example sys.stderr.write()